arch/src/main/res/layout/text.xml

```diff
@@ -4,6 +4,10 @@
 
     <data>
 
+        <variable
+            name="fm"
+            type="androidx.fragment.app.FragmentManager" />
+
```

Comment:
Extract to another layout ... maybe provide generic fragment container layout for `Activity`

Reply:
done.